.github/workflows/release_dev.yml

```
@@ -126,7 +126,7 @@ jobs:
         name: Create a release and upload the build
         uses: softprops/action-gh-release@v2.2.2
         with:
-          name: ChapterMaster ${{ needs.prepare_release.outputs.tag_name }}
+          name: ChapterMaster ${{ needs.prepare_release.outputs.tag_name | replace('/', '-') }}
```

actionlint (1.7.4)

got unexpected character ' ' while lexing || operator, expecting '|'

(expression)